### DISCUSSION TITLE
Is it possible to skip creation of "-state" folders/files?

### DISCUSSION THREAD

**Discussion by choowkee** [kohya-ss/musubi-tuner#423] [Q&A] [answered]
Hi, I am training my first WAN 2.1 lora with musubi and so far its smooth sailing.

But I noticed that every time I save an epoch a "-state" folder gets created with some files in it. Each folder is closing in on 1gb of size and thats a problem for me since I use rented cloud gpus so disk space is a consideration for me.

Are these files strictily necessary for ongoing training? Or can they be delated once an epoch has been saved?

This is what I am talking about:

<img width="568" height="212" alt="image" src="https://github.com/user-attachments/assets/ea28def2-8839-4554-8edb-ca34adb5f5dd" />

Launch commands I used:

accelerate launch --num_cpu_threads_per_process 1 --mixed_precision fp16 wan_train_network.py ^
    --task t2v-14B ^
    --dit /workspace/model/base/wan2.1_t2v_14B_fp16.safetensors ^
	--vae /workspace/model/vae/wan_2.1_vae.safetensors ^
	--t5 /workspace/model/te/models_t5_umt5-xxl-enc-bf16.pth ^
	--sdpa ^
	--mixed_precision fp16 ^
	--fp8_base ^
	--fp8_scaled ^
	--fp8_t5 ^
	--dataset_config /workspace/dataset/dataset.toml ^
    --optimizer_type adamw8bit ^
	--learning_rate 1e-4 ^
	--gradient_checkpointing ^
    --max_data_loader_n_workers 2 ^
	--persistent_data_loader_workers ^
    --network_module networks.lora_wan ^
	--network_dim 32 ^
	--network_alpha 32 ^
    --timestep_sampling shift ^
	--discrete_flow_shift 3.0 ^
	--seed 2025 ^
    --output_dir /workspace/output ^
	--output_name SeiV7Tuner ^
	--log_config ^
	--logging_dir /workspace/logs ^
	--save_state ^
	--max_train_steps 2100 ^
	--save_every_n_steps 100

**Comment by choowkee** [comment, marked as answer]
Eh nevermind I didnt notice I had --save_state added in the commands :D
